### ISSUE TITLE
kitti/middlebury/eth3d数据集的 train 和 finetune 学习率问题

### ISSUE THREAD

**Issue by jieLin-world** [ZYangChen/MoCha-Stereo#25]
作者你好，请问你在针对 kitti/middlebury/eth3d 这三个数据集进行 train 和 finetune 的时候学习率分别设置为多少，是所有实验的学习率都设置为 2e -4，还是设置成其他值？我按照您论文中的设置，使用混合数据集进行 train 和finetune的时侯学习率都设置为 2e -4，出现了梯度爆炸的问题。
![log](https://github.com/user-attachments/assets/3b1b72dd-525e-4b02-84be-d67b898bc469)

**Comment by ZYangChen** [comment]
都是2e-4。我在我的设备上无法复现您的这一问题，请问您是否有更改算法中的代码？

**Comment by ZYangChen** [comment]
[IGEV++](https://github.com/gangweix/IGEV-plusplus)这个工作复现并解决了您的问题，真厉害！

> NaN values during training: If you encounter NaN values in your training, this is likely due to overflow when using float16. This can happen when large gradients or high activation values exceed the range represented by float16.

**Comment by jieLin-world** [comment]
太好了，谢谢您^_^
